monocle/utils.py

```diff
@@ -241,6 +242,6 @@ def getter(attr, default):
         "task_severity",
         "task_type",
     ):
-        if params[sp]:
+        if params.get(sp):
```

Comment:
once the search endpoint is defined with protobuf, we can drop the the "," splitting business.